src/top_model/main.cpp

```diff
@@ -33,184 +33,215 @@ using TIME = NDTime;
-};
-cadmium::dynamic::modeling::ICs ics_TOP = {
-  cadmium::dynamic::translate::make_IC<iestream_input_defs<Message_t>::out,inp_control>("generator_con","ABPSimulator")
-};
-std::shared_ptr<cadmium::dynamic::modeling::coupled<TIME>> TOP = std::make_shared<cadmium::dynamic::modeling::coupled<TIME>>(
- "TOP", 
- submodels_TOP, 
- iports_TOP, 
- oports_TOP, 
- eics_TOP, 
- eocs_TOP, 
- ics_TOP 
-  );
-
-///****************////
-
-    auto elapsed1 = std::chrono::duration_cast<std::chrono::duration<double, std::ratio<1>>>(hclock::now() - start).count();
+    using info = cadmium::logger::logger<cadmium::logger::logger_info, 
+                 cadmium::dynamic::logger::formatter<TIME>, 
+                 oss_sink_provider>;
+    using debug = cadmium::logger::logger<cadmium::logger::logger_debug, 
+                  cadmium::dynamic::logger::formatter<TIME>, 
+                  oss_sink_provider>;
+    using state = cadmium::logger::logger<cadmium::logger::logger_state, 
+                  cadmium::dynamic::logger::formatter<TIME>, 
+                  oss_sink_provider>;
+    using log_messages = cadmium::logger::logger<cadmium::logger::logger_messages, 
+                         cadmium::dynamic::logger::formatter<TIME>, 
+                         oss_sink_provider>;
+    using routing = cadmium::logger::logger<cadmium::logger::logger_message_routing, 
+                    cadmium::dynamic::logger::formatter<TIME>, 
+                    oss_sink_provider>;
+    using global_time = cadmium::logger::logger<cadmium::logger::logger_global_time, 
+                        cadmium::dynamic::logger::formatter<TIME>, 
+                        oss_sink_provider>;
+    using local_time = cadmium::logger::logger<cadmium::logger::logger_local_time, 
+                       cadmium::dynamic::logger::formatter<TIME>, 
+                       oss_sink_provider>;
+    using log_all = cadmium::logger::multilogger<info, 
+                                                 debug, 
+                                                 state, 
+                                                 log_messages, 
+                                                 routing, 
+                                                 global_time, 
+                                                 local_time>;
+    using logger_top = cadmium::logger::multilogger<log_messages, global_time>;
+
+
+    /*******************************************/
+    /********************************************/
+    /****** APPLICATION GENERATOR *******************/
+    /********************************************/
+    string input_data_control = argv[1];
+    const char * i_input_data_control = input_data_control.c_str();
+
+    std::shared_ptr<cadmium::dynamic::modeling::model> generator_con = 
+                                                                       cadmium::dynamic::translate::make_dynamic_atomic_model<ApplicationGen, 

```

Comment:
Its going outside the screen even in git hub .You need to make it smaller

Reply:
Also to the next lines

Reply:
resolved as per suggestion